### PR TITLE
Feature/add versioning desc

### PR DESCRIPTION
**Pull request by GROwen**
# Checklist
- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated.
- [ ] Changelog entry has been written

Explain the **details** for making this change. What existing problem does the pull request solve?

# Changelog Entry

# Closing issues
Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: bay/images/docker/settings.php [hunk 1]
```diff
@@ -77,6 +77,9 @@
 // Set Stage File Proxy to use hotlink platform-wide
 $config['stage_file_proxy.settings']['hotlink'] = TRUE;
 
+// Skip unprotected files error.
+$settings['skip_permissions_hardening'] = TRUE;
```
Comment (nicksantamaria) [inline bay/images/docker/settings.php]: did you branch of Nadims PR?

Comment (GROwen) [reply bay/images/docker/settings.php]: fix incoming

File: README.md [hunk 2]
```diff
@@ -42,17 +42,50 @@ In your CDN configuration, set the header defined in `BAY_INGRESS_HEADER` with t
 
 Once deployed, if the header is missing in the request nginx will return a `405 Not Allowed` HTTP response.
 
-# Contribute
+## Contribute
 [Open an issue](https://github.com/dpc-sdp/bay) on GitHub or submit a pull request with suggested changes.
 
-# Support
+## Image versioning and release management
+Bay image versioning follows an abstracted semantic versioning model.
```
Comment (nicksantamaria) [inline README.md]:
```suggestion
Bay images use the following versioning model, tied to git branches of the same name.
```

File: README.md [hunk 3]
```diff
@@ -42,17 +42,50 @@ In your CDN configuration, set the header defined in `BAY_INGRESS_HEADER` with t
...
+A major version is identified with a numeral and the minor version is abstracted to `x`. This provides flexibility for 
+future versioning requirements.
+
+In practice this looks like `1.x`, `2.x` etc.
+
+At any one time the two most recent tags are supported. These are identified based on numerical order.
+
+The lower value tag references a 'production safe' image.
+
+The higher value tag includes major updates to the image in testing.
```
Comment (steveworley) [inline README.md:60]: I think this is more to represent backwards breaking changes. In testing implies that you shouldn't move to the higher number until the next higher number is released. 

In the example; 1.x would only receive security updates and 2.x would receive non-breaking feature updates. Projects should aim to run the latest version available.

Comment (GROwen) [reply README.md:60]: >Projects should aim to run the latest version available.

👍 
When will we create the x.x-dev branches? And then the next major version?

**Comment by nicksantamaria** [comment]
Closing as outdated

**Pull request closed without merging**